webdev3.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Greatest Leaflet Map Ever</title>
+   <!-- External Stylesheets -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css" />
+      <style>
+           
+/* Allow Map to fill page*/
+/* source: http://bl.ocks.org/d3noob/7654694 */
+/* CSS adjusted to display map coordinates source: https://kristofj-umd.github.io/webdev-f19/assignments/webdev3.html*/
+        body {
+            padding: 0;
+            margin: 0;
+                }
+        html, body, #map {
+            height: calc(100vh - 60px);
+            width: 100%;
+                }
+           map {
+               top: 60px;
+                }
+/*Stylize Header*/
+           header {
+               top: 0;
+               width: 100%;
+               z-index: 999;
+                position: fixed;
+                height: 60px;
+               background-color: maroon;
+               color: gold;
+               Padding-left: 12px;
+               font-size: 15px;
+               overflow: hidden;
+               font-family: "Times New Roman", Times, serif;
+                }
+/* Display text inline*/
+           header * {            
+               Display: inline-block;
+               margin-right: 20px;
+               margin-top: 10px;
+                }
+           .right {
+               float: right;
+           }
+/* Style Coordinates */
+          #mapCoordinates {
+              color: blue;
+              z-index: 999999;
+              position: relative;
+              margin: 20px;
+          }
```

Comment:
Great work getting this to show up without a scrollbar appearing along the side.